changelog/README.md

```diff
@@ -1,5 +1,14 @@
 # Changelog
 
+## 21st February 2025
```

Comment:
I want to follow the same conventions that we've established in Connector API for Changelog entries:
1. We put the affected API Operation first
2. We are precise about exactly what has changed
3. Links should have the same name as the thing they are linking to (and the links should work)